nixos/modules/system/boot/networkd.nix

```diff
@@ -1900,6 +1900,9 @@ in
     };
 
     systemd.network.wait-online = {
+      enable = mkEnableOption (lib.mdDoc "systemd-networkd-wait-online") // {
```

Comment:
Can you add some description here, describing when one wants to set this option, similar to how it's in the PR description?

Reply:
[How's this?](https://github.com/NixOS/nixpkgs/compare/e2b7e692f5a618a1593ac08c8ce4c3711e7408d2..51b5602b458f8ab2b62f397fe98dcd16b7f44135)